os/drivers/pm/pm.c

```diff
@@ -0,0 +1,204 @@
+/****************************************************************************
+ *
+ * Copyright 2024 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+#include <tinyara/config.h>
+
+#include <tinyara/fs/fs.h>
+#include <tinyara/fs/ioctl.h>
+#include <tinyara/pm/pm.h>
+#include <tinyara/clock.h>
+
+#include <errno.h>
+#include <debug.h>
+#include <assert.h>
+
+/* Added to record timer countdown interrupt */
+struct pm_timer_s g_pm_timer = {0, 0, 0};
```

Comment:
It looks that pm.c file uses only. Then, let's add `static`.

Reply:
Hi @sunghan-chang , I think os/arch/arm/src/amebasmart/amebasmart_pmhelpers.c and os/pm/pm_activity.c is still using this.

Reply:
yes , it is a global variable